contracts/protocol/clients/FermionFNFTPriceManager.sol

```diff
@@ -403,15 +430,19 @@ contract FermionFNFTPriceManager is FermionErrors, ERC2771Context, IFermionFNFTP
             return;
         }
 
-        FermionTypes.PriceUpdateProposal storage proposal = Common
-            ._getBuyoutAuctionStorage(currentEpoch)
-            .currentProposal;
+        FermionTypes.BuyoutAuctionStorage storage $ = Common._getBuyoutAuctionStorage(currentEpoch);
+        uint256 voterProposalId = $.voters[from].proposalId;
 
-        if (proposal.state != FermionTypes.PriceUpdateProposalState.Active) {
+        // check if no proposal has been created yet
+        if (voterProposalId >= $.priceUpdateProposals.length) {
             return;
         }
 
-        FermionTypes.PriceUpdateVoter storage voter = proposal.voters[from];
+        FermionTypes.PriceUpdateProposal storage proposal = $.priceUpdateProposals[voterProposalId];
+
+        if (proposal.state != FermionTypes.PriceUpdateProposalState.Active) return;
+
+        FermionTypes.PriceUpdateVoter storage voter = $.voters[from];
```

Comment:
Move this to line 434, and use it already in `uint256 voterProposalId = $.voters[from].proposalId;`